internal/ddt/node_doc_transformer_test.go

```diff
@@ -37,14 +37,15 @@ func TestNodeLinkAbsolute(t *testing.T) {
 
 func TestTransformNodeLinkRelative(t *testing.T) {
 	get := func(url string) (bool, *Node, error) {
-		if url == "foo/bar/baz" {
+		if url == "foo/bar/baz" || url == "foo/bar" {
 			return true, &Node{root: "/tmp/xyz", Path: filepath.Join("/tmp/xyz", url)}, nil
 		}
 		return false, &Node{}, nil
 	}
 	dt, _ := NewNodeDocTransformer("/tree", "foo/bar", get, "test")
 
 	expected := map[string]string{
+		"<a href=\"./\"></a>":         "<a href=\"/tree/foo/bar?v=test\" data-node=\"foo/bar\"></a>",
```

Comment:
When using `href=""` is also accepted, please add an additional test for that.

Reply:
When using `href="."` is also accepted, please add an additional test for that.

Reply:
👍